packages/e2e-tests/run.ts

```diff
@@ -16,6 +16,24 @@ const publishScriptNodeVersion = process.env.E2E_TEST_PUBLISH_SCRIPT_NODE_VERSIO
 const DEFAULT_BUILD_TIMEOUT_SECONDS = 60;
 const DEFAULT_TEST_TIMEOUT_SECONDS = 60;
 
+if (!process.env.E2E_TEST_AUTH_TOKEN) {
+  console.log(
+    "No auth token configured! Please configure the E2E_TEST_AUTH_TOKEN environment variable with an auth token that has the scope 'project:read'!",
+  );
+}
+
+if (!process.env.E2E_TEST_DSN) {
+  console.log('No DSN configured! Please configure the E2E_TEST_DSN environment variable with a DSN!');
+}
+
+if (!process.env.E2E_TEST_AUTH_TOKEN || !process.env.E2E_TEST_DSN) {
+  process.exit(1);
+}
+
+const envVarsToInject = {
+  REACT_APP_E2E_TEST_DSN: process.env.E2E_TEST_DSN,
```

Comment:
Why do we do the renaming here? Shouldn't it just be called `E2E_TEST_DSN` everywhere?

Reply:
Since CRA apps are static we need to inject the DSN during build time (I guess that's more or less only possible with env vars) and CRA has restrictions on what env vars may look like: https://create-react-app.dev/docs/adding-custom-environment-variables/ (They only allow env vars prefixed with `REACT_APP_`)